jsonp-tck/pom.xml

```diff
@@ -48,149 +54,111 @@
+            <groupId>org.eclipse.parsson</groupId>
+            <artifactId>parsson-media</artifactId>
+            <version>1.1.0</version>
             <scope>test</scope>
-            <version>${payara.arquillian.container.version}</version>
-        </dependency>
-
-        <!-- jdk11+ dependencies -->
-        <dependency>
-            <groupId>jakarta.xml.bind</groupId>
-            <artifactId>jakarta.xml.bind-api</artifactId>
-            <version>3.0.1</version>
-        </dependency>
-        <dependency>
-            <groupId>jakarta.activation</groupId>
-            <artifactId>jakarta.activation-api</artifactId>
-            <version>2.1.0</version>
         </dependency>
     </dependencies>
 
     <build>
         <plugins>
-            <plugin>
-                <groupId>org.apache.maven.plugins</groupId>
-                <artifactId>maven-dependency-plugin</artifactId>
-                <version>3.2.0</version>
-                <executions>
-                    <execution>
-                        <id>copy</id>
-                        <phase>package</phase>
-                        <goals>
-                            <goal>copy</goal>
-                        </goals>
-                    </execution>
-                    <execution>
-                        <id>copy-dependency</id>
-                        <phase>package</phase>
-                        <goals>
-                            <goal>copy-dependency</goal>
-                        </goals>
-                    </execution>
-                </executions>
-                <configuration>
-                    <artifactItems>
-                        <artifactItem>
-                            <groupId>org.apache.derby</groupId>
-                            <artifactId>derby</artifactId>
-                            <version>10.15.2.0</version>
-                        </artifactItem>
-                        <artifactItem>
-                            <groupId>org.apache.derby</groupId>
-                            <artifactId>derbytools</artifactId>
-                            <version>10.15.2.0</version>
-                        </artifactItem>
-                        <artifactItem>
-                            <groupId>org.apache.derby</groupId>
-                            <artifactId>derbyshared</artifactId>
-                            <version>10.15.2.0</version>
-                        </artifactItem>
-                    </artifactItems>
-                    <outputDirectory>${project.build.directory}/server-dependencies</outputDirectory>
-                </configuration>
-            </plugin>
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-surefire-plugin</artifactId>
                 <version>3.0.0-M5</version>
                 <configuration>
+                    <excludes>
+                        <exclude>JSONPSigTest*</exclude>
```

Comment:
Just clarifying - should we be excluding this?
Is it just for standalone runs or are we challenging it?

Reply:
For standalone run, it excludes the `JSONPSigTest` from `dependencyToScan` test through junit extension `META-INF/services/org.junit.jupiter.api.extension.Extension` and run it separately via junit test class in `fish.payara.server.tck.jsonp.signaturetest.JsonpSignatureTest` with arquillian extension.

The reason is I have an issue with the signature test required a system property definition such `signature.sigTestClasspath` and `jimage.dir` that junit extension `META-INF/services/org.junit.jupiter.api.extension.Extension` cannot load.

So it is an alternative solution to be able to read system property and execute the signature test.